forge-game/src/main/java/forge/game/ability/effects/RestartGameEffect.java

```diff
@@ -70,6 +70,7 @@ public void resolve(SpellAbility sa) {
             p.setLifeLostLastTurn(0);
             p.resetCommanderStats();
             p.resetCompletedDungeons();
+            p.resetRingTemptedYou();
```

Comment:
does this also need to reset the ring bearer and the ring effect?

Reply:
Looks like the next line does that.